corehq/apps/app_manager/views/download.py

```diff
@@ -291,7 +291,7 @@ def create_build_files_if_necessary_handling_conflicts(is_retry=False):
             payload = request.app.fetch_attachment(full_path)
         except ResourceNotFound:
             if not build_profile_id:
-                create_build_files()
+                create_build_files_if_necessary_handling_conflicts()
```

Comment:
nit: are you intentionally not combining this and the following elif?

Reply:
Fixed it, thanks.